site/content/en/docs/concepts/admission_fair_sharing.md

```diff
@@ -0,0 +1,95 @@
+---
+title: "Admission Fair Sharing"
+date: 2025-05-28
+weight: 6
+description: >
+  A mechanism for ordering workloads based on the historical resource usage of their source LocalQueues, giving preference to those that have consumed fewer resources over time.
+---
+
+# Admission Fair Sharing
+
+Admission Fair Sharing is an alpha feature introduced in Kueue v0.12.0 that helps distribute resources fairly between multiple LocalQueues targeting the same ClusterQueue. It orders workloads based on the historical resource usage of their source LocalQueues, giving preference to those that have consumed less resources over time.
+
+## How it works
+
+When multiple workloads compete for resources within a ClusterQueue:
+
+1. Kueue tracks resource usage history for each LocalQueue
+2. Workloads from LocalQueues with lower historical usage get admitted before those from high-usage queues
+3. Usage values decay over time based on configurable parameters
+
+## Configuration
+
+
+### Feature Gate
+
+Admission Fair Sharing is an alpha feature, in order to use it a feature gate must be enabled. You can enable it by:
+
+1. Setting the feature gate flag when starting kueue-controller:
+```bash
+--feature-gates=AdmissionFairSharing=true
+```
```

Comment:
Instead of this one, could you use a card like https://kueue.sigs.k8s.io/docs/concepts/multikueue/ and avoid describing featuregate enabling way?